esp/src/eclwatch/templates/WUDetailsWidget.html

```diff
@@ -205,7 +205,7 @@ <h2>
                     <input id="${id}EndDateTime" name="LogFilter_AbsoluteTimeRange_EndDate" type="hidden" />
                     <input id="${id}EndDate" title="${i18n.ToDate}:" name="EndDate" data-dojo-props="trim: true" required="required" data-dojo-type="dijit.form.DateTextBox" />
                     <input id="${id}EndTime" title="" name="EndTime" data-dojo-props="trim: true" value="T19:30:00" data-dojo-type="dijit.form.TimeTextBox" />
-                    <input id="${id}RelativeTimeRangeBuffer" title="${i18n.RelativeTimeRange}" name="LogFilter_RelativeTimeRangeBuffer" colspan="2" data-dojo-type="dijit.form.TextBox" />
+                    <input id="${id}RelativeTimeRangeBuffer" title="${i18n.RelativeTimeRange}" name="LogFilter_RelativeTimeRangeBuffer" value="43200" colspan="2" data-dojo-type="dijit.form.TextBox" />
```

Comment:
What is 43200 representing here?

Reply:
That's the default that ESP sets, got added to the v9 UI a couple weeks ago, but not to the v5 version (added a comment on that PR, https://github.com/hpcc-systems/HPCC-Platform/pull/19216/files)